entry/src/main/js/default/pages/index/index.js

```diff
@@ -127,4 +140,75 @@ export default {
     onWeekClick: function() {
         console.log("Week Button clicked !");
     },
+    fetchData : function(){
+        let data;
+        fetch.fetch({
+            url:'https://neumorphism-api.herokuapp.com/neumorphism/living_room',
+            responseType:"json",
+            method: 'GET',
+            success:function(resp)
+            {
+                data = JSON.parse(resp.data);
+            },
+            fail:(err,code) => {
+                console.log("fail data:"+ JSON.stringify(err));
+                console.log("fail code:"+ code)
+            },
+            complete: ()=>{
+                this.username = data.username;
+                this.living_room_active = data.living_room_active;
+                this.dining_room_active = data.dining_room_active;
+                this.bed_room_active = data.bed_room_active;
+                this.bath_room_active = data.bath_room_active;
+                this.backyard_room_active = data.backyard_room_active;
+                this.living_room = data.living_room;
+                this.smart_tv = data.smart_tv;
+                this.ac = data.ac;
+                this.washing_machine = data.washing_machine;
+                this.refrigerator = data.refrigerator;
+                this.carousel = data.carousel;
+                this.dry_mode = data.dry_mode;
+                this.cool_mode = data.cool_mode;
+                this.fan_mode = data.fan_mode;
+                this.turbo_mode = data.turbo_mode;
+                this.lamp1_p = data.lamp1_p;
+                this.lamp2_p = data.lamp2_p;
+                this.fan1_p = data.fan1_p;
+                this.fan2_p = data.fan2_p;
+                this.temperature = data.temperature;
+                this.humidity = data.humidity;
+                this.location = data.location;
+                this.alert_title = data.alert_title;
+                this.alert_desc = data.alert_desc;
+            }
+        })
+        fetch.fetch({
+            url:'https://weatherdbi.herokuapp.com/data/weather/bangalore',
+            responseType:"json",
+            method: 'GET',
+            success:function(resp)
+            {
+                data = JSON.parse(resp.data);
+            },
+            fail:(err,code) => {
+                console.log("fail data:"+ JSON.stringify(err));
+                console.log("fail code:"+ code)
+            },
+            complete: ()=>{
+                this.temperatureInCelsius = data.currentConditions.temp.c;
+                this.weather = data.currentConditions.comment;
+            }
+        })
+        const date = new Date();
+        const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun",
```

Comment:
please take the values of these arrays from locale string files

Reply:
Actually this is intentional and works for both chinese and english language.
To get the name of current month :

1. We use `date.getMonth()` which returns the current month in number i.e. 0, 1, 2 and so on.
2. We use this number to get the corresponding month name from `monthNames` array.
3. Then we convert it to chinese or english accordingly using `this.$t('strings.'+monthNames[date.getMonth()])`